roles/nv_gpu_install_from_commit/tasks/main.yml

```diff
@@ -1,4 +1,15 @@
----
```

Comment:
I like the 3 dots at the beginning of a yaml , but is a personal pref, is ok not to have them

Reply:
hum, I don't really mind for ansible scripts, so I put it back ;-)